src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope.c

```diff
@@ -1850,113 +2172,117 @@ s16 func_80823A0C(PlayState* play, Vtx* vtx, s16 arg2, s16 arg3) {
     s16* ptr2;
     s16* ptr3;
     s16* ptr4;
-    s16 phi_s2;
-    s16 phi_t0;
-    s16 phi_a1;
-    s16 phi_a2;
-    s16 phi_t3;
-    s16 phi_t1;
+    s16 bufIAfterPageSections;
+    s16 pageBgQuadX;
+    s16 pageBgQuadY;
+    s16 i;
+    s16 j;
+    s16 bufI;
```

Comment:
not a big fan of `bufI` but after looking at the usage.... guh. i can't think of anything meaningfully better to suggest

Reply:
it's the next free index in the vertex buffer :^)